src/UNL/PersonInfo.php

```diff
@@ -450,7 +451,14 @@ public static function getURL($additional_params = [])
      */
     public function determineView()
```

Comment:
I don't think we need this function since the default value is 'instructions' and the view is provided by the htaccess file

Reply:
are you talking about getURL() function or the determineView()? Errors pop up when the determineView() function is removed.